spring-sample/docs/modules/ROOT/pages/spring-boot-client.adoc

```diff
@@ -0,0 +1,139 @@
+= Create a Spring Boot Application for {hazelcast-cloud}
+:description: In this tutorial, you'll learn how to create a basic Spring Boot application that connects to a Serverless cluster on {hazelcast-cloud}.
+:page-product: cloud
+:page-layout: tutorial
+:page-categories: Spring Boot, Get Started
+:page-lang: java
+:page-est-time: 10 mins
+:page-serverless: true
+:url-spring-boot: https://docs.spring.io/spring-boot/docs/current/reference/htmlsingle/#getting-started
+:url-spring-hazelcast: https://docs.spring.io/spring-boot/docs/current/reference/htmlsingle/#io.hazelcast
+:github-directory: https://github.com/hazelcast/hazelcast-cloud-code-samples/tree/master/spring-sample 
+
+{description}
+
+== Context
+
+Spring Boot helps you to create stand-alone, production-grade Spring-based applications. If you're unfamiliar with Spring or Spring Boot, see the link:{url-spring-boot}[Spring Boot documentation].
```

Comment:
Maybe too many compound adjectives? Would it be clearer to say this, or something similar.

....stand-alone Spring-based applications that are production grade.

Reply:
I agree. I actually took this from the Spring Boot docs. We can reword it.